pyproject.toml

```diff
@@ -123,3 +123,16 @@ target-version = ['py38']
 [tool.check-wheel-contents]
 # W009: Wheel contains multiple toplevel library entries
 ignore = "W009"
+
+[tool.ruff]
+src = ["pytest"]
```

Comment:
I think this should be `"src"` if I understand https://docs.astral.sh/ruff/settings/#src correctly.

Reply:
For sure, my bad.